hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMHAConfiguration.java

```diff
@@ -114,6 +114,14 @@
   )
   private int raftLogPurgeGap = 1000000;
 
+  @Config(key = "ratis.snapshot.threshold",
+      type = ConfigType.LONG,
+      defaultValue = "100000L",
+      tags = {SCM, OZONE, HA, RATIS},
+      description = "The threshold to trigger a Ratis taking snapshot " +
+          "operation")
```

Comment:
operation -> operation for SCM.

Reply:
seems not a typo?

Reply:
Not a typo, the description indicates that this config is for SCM.

"The threshold to trigger a Ratis taking snapshot operation" -> "The threshold to trigger a Ratis taking snapshot operation for SCM StateMachine" 
BTW, please change the `defaultValue` too.